biz.aQute.bnd.maven/src/aQute/bnd/maven/lib/resolve/BndrunContainer.java

```diff
@@ -1,12 +1,13 @@
 package aQute.bnd.maven.lib.resolve;
 
+import static java.util.Optional.ofNullable;
```

Comment:
I am not a fan of this static import. `ofNullable` in code is too ambiguous. In Java 9, Stream has the same static method name for example.